terasoluna-tourreservation-domain/src/main/resources/META-INF/spring/terasoluna-tourreservation-codelist.xml

```diff
@@ -7,6 +7,10 @@
            http://www.springframework.org/schema/task http://www.springframework.org/schema/task/spring-task.xsd
        ">
 
+  <bean id="fallbackToLocale" abstract="true">
```

Comment:
- [ ] Abstract parent bean should be as follows::

```xml
<bean id="abstractSimpleI18nCodeList"
    class="org.terasoluna.gfw.common.codelist.i18n.SimpleI18nCodeList"
    abstract="true">
    <property name="fallbackTo" value="en" />
</bean>
```

Because...
* bean class should be specified on parent.
* bean id should express it's abstract.

And `AbstractJdbcCodeList` should be `abstractJdbcCodeList` (lower camel case) too.

Reply:
Modified by 62277e151b2b687e530298d4d4d90904b6ba62c3
We also modified `AbstractJdbcCodeList` at the same time.